go.mod

```diff
@@ -40,50 +40,101 @@ require (
 )
 
 require (
+	github.com/BurntSushi/toml v1.3.2 // indirect
```

Comment:
You issue does not suppose any changes in the dependencies, so go.mod and go.sum should not be modified.

Reply:
this was generated from go.mod